components/nodes/node-stanford-news.tsx

```diff
@@ -59,38 +100,69 @@ export const NodeStanfordNews = ({node, ...props}: NewsNodeProps) => {
 }
 
 export const NodeStanfordNewsListItem = ({node, ...props}: NewsNodeProps) => {
+
   return (
-    <article className="su-rs-px-2 su-rs-pt-2 su-rs-pb-4" {...props}>
-      <DrupalLink href={node.path.alias}>
-        <h2 className="su-text-cardinal-red">{node.title}</h2>
-      </DrupalLink>
+    <article {...props}>
+      <div className="su-text-18 su-mb-14">
+        {node.su_news_publishing_date && <>{formatDate(node.su_news_publishing_date + ' 12:00:00')}</>}
+      </div>
+      <div className="su-grid su-grid-cols-[65%_35%] su-gap-1">
+        <div>
+          <DrupalLink className="su-text-digital-red su-no-underline hover:su-underline" href={node.path.alias}>
+            <h2 className="su-type-2">{node.title}</h2>
+          </DrupalLink>
+          {node.su_news_dek && <div className="su-rs-mb-1">{node.su_news_dek}</div>}
+        </div>
+        <div className="su-hidden lg:su-block">
+          {node?.su_news_featured_media?.field_media_image &&
+            <DrupalImage
+                src={node.su_news_featured_media.field_media_image.uri.url}
+                alt={node.su_news_featured_media.field_media_image.resourceIdObjMeta.alt}
+                height={node.su_news_featured_media.field_media_image.resourceIdObjMeta.height}
+                width={node.su_news_featured_media.field_media_image.resourceIdObjMeta.width}
+            />
+          }
+        </div>
+      </div>
+      {node.su_news_topics && node.su_news_topics.map(cardTopic =>
+        <div key={cardTopic.id} className="su-mt-10">
+          {cardTopic.name}
+        </div>
+      )}
     </article>
   )
 }
 
 export const NodeStanfordNewsCard = ({node, ...props}: NewsNodeProps) => {
 
   return (
-    <article className="su-shadow-lg su-text-15 su-rs-px-2 su-rs-pt-2 su-rs-pb-4" {...props}>
-      {node?.su_news_featured_media?.field_media_image &&
-          <DrupalImage
-              src={node.su_news_featured_media.field_media_image.uri.url}
-              alt={node.su_news_featured_media.field_media_image.resourceIdObjMeta.alt}
-              height={node.su_news_featured_media.field_media_image.resourceIdObjMeta.height}
-              width={node.su_news_featured_media.field_media_image.resourceIdObjMeta.width}
-          />
-      }
-
-      <DrupalLink href={node.path.alias}
-                  className="su-no-underline su-text-cardinal-red hover:su-underline hover:su-text-black">
-        <h2>
-          {node.title}
-        </h2>
-
-        {node.su_news_topics && node.su_news_topics.map(topic =>
+    <article {...props}>
+      <Card
+        image={
```

Comment:
`image` should be null if no image is available. Verify you don't get an empty space in the cards. You might take the basic page as an example: https://github.com/SU-SWS/sulgryphon-nextjs/blob/1.x/components/nodes/node-stanford-page.tsx#L88